File: src/main/java/hu/gerviba/webschop/dao/OpeningEntityDto.java
package hu.gerviba.webschop.dao;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class OpeningEntityDto {
    
    private int timeIntervals = 4;
    private String dateStart;
    private String dateEnd;
    private String orderStart;
    private String orderEnd;
    private String feeling;
    private int maxOrder = 120;
    private int maxOrderPerInterval = 15;
    private int maxExtraPerInterval = 0;
    private int intervalLength = 30;

    private int maxAlpha = 0;
    private int maxBeta = 0;
    private int maxGamma = 0;
    private int maxDelta = 0;
    private int maxLambda = 0;
    
}

File: src/main/kotlin/hu/kirdev/schpincer/dto/OpeningEntityDto.kt
package hu.gerviba.webschop.dto

data class OpeningEntityDto(
    var timeIntervals: Int = 4,

    val dateStart: String = "",
    val dateEnd: String = "",
    val orderStart: String = "",
    val orderEnd: String = "",
    val feeling: String = "",

    var maxOrder: Int = 120,
    var maxOrderPerInterval: Int = 15,
    var maxExtraPerInterval: Int = 0,
    var intervalLength: Int = 30,

    var maxAlpha: Int = 0,
    var maxBeta: Int = 0,
    var maxGamma: Int = 0,
    var maxDelta: Int = 0,
    var maxLambda: Int = 0
)

